### ISSUE TITLE
The Downloads label does not appear in the gallery section.

### ISSUE THREAD

**Issue by dagosalas** [pkp/pkp-lib#7053]
**Describe the bug**
The Downloads label does not appear in the gallery section. This is very confusing for screen readers and aesthetically the lack of that word is noticeable. The class assigned to h2 is incorrect, it currently has _class="pkp_screen_reader"_ (which makes it disappear) and the correct class should be _class="label"_ (which makes the word Downloads show).

**To Reproduce**
1. View the detail of an article and look at the gallery section. (which is where the download formats are) and the word Download does not appear as a title or label. And in the other components it does.

**What application are you using?**
 OJS, OMP 3.3.0-5 y 3.3.0-6

**Additional information**
 Screenshots:
![Screen Shot 2021-05-09 at 22 59 49](https://user-images.githubusercontent.com/21068555/117853939-9744e500-b24e-11eb-8c9f-ebab3b141bce.png)
![Screen Shot 2021-05-09 at 22 59 12](https://user-images.githubusercontent.com/21068555/117853965-9e6bf300-b24e-11eb-9298-532bc917d2d5.png)

And with the right class:

![Screen Shot 2021-05-09 at 23 00 12](https://user-images.githubusercontent.com/21068555/117854053-b5124a00-b24e-11eb-9a52-f146f1e6377f.png)
![Screen Shot 2021-05-09 at 23 00 04](https://user-images.githubusercontent.com/21068555/117854065-b93e6780-b24e-11eb-81e0-7dd05931b70d.png)

**Comment by NateWr** [comment]
Hi @dagosalas, this is a deliberate design decision. The belief is that the visual prominence of the button along with the label is sufficient for a sighted user to identify them as download links. The `<h2>` with the downloads label is present to help readers using assistive technology navigate to the download links.

I'm going to close this for now, as we don't plan to change it, but I'll keep an eye out for any problems this might cause in our recorded user testing sessions.
